src/int.cr

```diff
@@ -502,6 +502,63 @@ struct Int
     io << type
   end
 
+  enum BinaryPrefixFormat
+    # The IEC standard prefixes (`Ki`, `Mi`, `Gi`, `Ti`, `Pi`, `Ei`, `Zi`, `Yi`)
+    # based on powers of 1000.
+    IEC
+
+    # Extended range of the JEDEC units (`K`, `M`, `G`, `T`, `P`, `E`, `Z`, `Y`) which equals to
+    # the prefixes of the SI system except for uppercase `K` and is based on
+    # powers of 1024.
+    JEDEC
+  end
+
+  # Prints this integer as a binary value in a human-readable format using
+  # a `BinaryPrefixFormat`.
+  #
+  # Values with binary measurements such as computer storage (e.g. RAM size) are
+  # typically expressed using unit prefixes based on 1024 (instead of multiples
+  # of 1000 as per SI standard). This method by default uses an extended range
+  # of the `BinaryPrefixFormat::JEDEC` units (`K`, `M`, `G`, `T`, `P`, `E`, `Z`,
+  # `Y`) which equals to the prefixes of the SI system except for uppercase `K`
+  # and is based on powers of 1024.
+  # The IEC standard prefixes (`Ki`, `Mi`, `Gi`, `Ti`, `Pi`, `Ei`, `Zi`, `Yi`)
+  # based on powers of 1000 will be used if *format* is `BinaryPrefixFormat::IEC`.
+  #
+  # ```
+  # 1.humanize_bytes                        # => "1B"
+  # 1024.humanize_bytes                     # => "1.0KB"
+  # 1536.humanize_bytes                     # => "1.5KB"
+  # 524288.humanize_bytes                   # => "512KB"
+  # 1073741824.humanize_bytes(format: :IEC) # => "1.0GiB"
+  # ```
+  #
+  # See `Number#humanize` for more details on the behaviour and arguments.
+  def humanize_bytes(io : IO, precision : Int = 3, separator = '.', *, significant : Bool = true, format : BinaryPrefixFormat = :JEDEC) : Nil
```

Comment:
`IEC` is still not the default.

Reply:
Having `BinaryPrefixFormat::IEC`/`BinaryPrefixFormat::JEDEC` may be better for the doc

Reply:
Ok, fine...